docs/development.md

```diff
@@ -148,6 +148,11 @@ You need a student account which can be created by instructors.
 1. Log out and log in to `http://localhost:8888/admin/adminSearchPage` as administrator.
 1. Search for the student you added in as instructor. From the search results, click anywhere on the desired row (except on the student name) to get the course join link for that student.
 1. Log out and use that join link (again, change the base URL to `http://localhost:8888` if necessary) to log in as a student.
+1. Masquerade mode can also be used to impersonate different users by adding user=username to the URL
+ e.g. `http://localhost:8888/page/studentHomePage?user=johnKent`.
+
+**NOTE**
+> Masquerade mode will only work if the logged in user has admin access.
```

Comment:
I think that it would be better to put this information under **As administrator**, then you do not need to write an additional note about having admin access.